pulp/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from .run_tests import *
```

Comment:
while you're at it, add a new line here.

Reply:
Same, modified to import only pulpTestAll.